src/res/topic.txt

```diff
@@ -12,3 +12,7 @@ What build system do you use?
 What compiler do you use, and why?
 What are your favorite resources for learning C++?
 What is the most difficult part of C++?
+What are your favorite C++ features?
+How long have you been learning C++?
```

Comment:
https://github.com/cppdiscord/bot/blob/5087bdd17014135e95df77b4bf764656afb69af3/src/res/topic.txt#L2 Your question is too similar to this, please remove it or replace line 2 with it